### DISCUSSION TITLE
E4 HelpService?

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-platform/.github#19] [Q&A] [answered]
Currently there is `org.eclipse.ui.help.IWorkbenchHelpSystem` but ti seems there is no E4 replacement for the help.

Should we introduce a E4 `HelpService` for that purpose?

**Comment by akurtakov** [comment]
Only *extra* requirement would be to have it used in some place in the workbench too so we are sure the new implementation works.

**Comment by vogella** [comment, marked as answer]
We have org.eclipse.e4.ui.services.help.EHelpService

**Reply by laeubi** [reply]
Thanks for the hint! It seems this service is missing from https://wiki.eclipse.org/Eclipse4/RCP/EAS/List_of_All_Provided_Services

**Reply by vogella** [reply]
Please add it. :-)
